### DISCUSSION TITLE
Eco Rules for CPP

### DISCUSSION THREAD

**Discussion by seenusankar** [green-code-initiative/ecoCode#223] [General]
Hello team, are there any plans to implement a plugin for CPP I mean eco-code plugin for CPP.

Regards,
Sreeni.

**Comment by glalloue** [comment]
Hi @seenusankar,
CPP is not currently in our roadmap.
but several companies and people asked us about this subject.
So we are opened to integrate new contributors who want to work on a CPP plugin.
If you want to work on it, do not hesitate to contact us and we will be able to help you starting.

**Comment by dedece35** [comment]
Hi @seenusankar,
I know the others plugins and I will be happy to help to create the plugin for CPP !!! :p
but I don't know a lot about CPP (I learned CPP at school a long time ago :p)
